src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -73,14 +73,14 @@ const TabbedLayout: FunctionComponent<TabbedLayoutProps> = ({
                 <Box display={ fixedHeight ? 'flex' :'block' } flexDirection="column" height={ fixedHeight ? 1 : 'auto' }>
                     <Box flexGrow={ 0 } flexShrink={ 0 }>
                         <Box display="flex" justifyContent="space-between">
-                            <Box className={ classes.breadcrumbs } p={ 2 } pl={ 3 }>
+                            <Box className={ classes.breadcrumbs } pt={ 3 } px={ 3 }>
```

Comment:
Do you think it's better to possibly putting a `p` on a parent box, and only dealing with the vertical spacing internally? We can discuss this if you like